tasks/version.js

```diff
@@ -73,7 +73,11 @@ module.exports = function(grunt) {
         literal = semver.valid(release);
 
     if ( bump && semver.valid(version) ) {
-      newVersion = semver.inc(version, release);
+      if (typeof options.prereleaseidentifier !== 'undefined') {
```

Comment:
Would you mind camel-casing this? So, `prereleaseIdentifier` … or _maybe_ `prereleaseId`? Your call, but I'd like it to be camel-cased to stick with convention.